tests/render.test.ts

```diff
@@ -23,3 +24,36 @@ test('render vite.config.js from template',  () => {
   expect(source).toMatch('viteCommonjs()')
   fs.rmdirSync(outDir, { recursive: true })
 });
+
+test('sort report by item.num', () => {
+  interface ConverObj {
+    num: string;// 'Number'
+    feat: string;// 'Conversion item'
+    times?: number;// 'Conversion times'
+  }
```

Comment:
Is it an interface that can be `import`ed from `report.ts`?

Reply:
### 50% / 50%, 
just like the `Vue3 composition` thought, keep it **tight**
But, it can imported from `report.ts` for the long journey.